spec/spec.md

```diff
@@ -331,7 +356,10 @@ where:
   - `uri`: (optional) a URI representing the web address associated with the dapp endpoint making this authorization request. If present, it must be an absolute, hierarchical URI.
   - `icon`: (optional) a relative path (from `uri`) to an image asset file of an icon identifying the dapp endpoint making this authorization request
   - `name`: (optional) the display name for this dapp endpoint
-- `cluster`: (optional) if set, the Solana network cluster with which the dapp endpoint intends to interact; supported values include `mainnet-beta`, `testnet`, `devnet`. If not set, defaults to `mainnet-beta`.
+- `chain`: (optional) if set, the [chain identifier](#chain-identifiers) for the chain with which the dapp endpoint intends to interact; supported values include `solana:mainnet`, `solana:testnet`, `solana:devnet`. If not set, defaults to `solana:mainnet`.
+- `auth_token`: (optional) an opaque string previously returned by a call to [`authorize`](#authorize), or [`clone_authorization`](#clone_authorization). When present, the wallet endpoint should attempt to reauthorize the dapp endpoint silently without prompting the user. 
+- `sign_in_payload`: (optional) a value object containing the payload portion of a [Sign In With Solana message](https://siws.web3auth.io/spec). If present, the wallet endpoint should present the SIWS message to the user and return the resulting signature and signed message, as described below.  
```

Comment:
Also, did we align our payload for this with Wallet Standard?

Reply:
Sort of. Wallet standard does not enforce a payload format, but states that "a wallet may modify the payload before signing, for example to add a message preamble". So wallet standard returns the signature and signed message, which may have been modified by the wallet.

Reply:
oh my mistake, I was referring to the `signMessages` returned payload. the sign in with solana payload is aligned with the [solana wallet standard extension](https://github.com/solana-labs/wallet-standard/blob/master/packages/core/features/src/signIn.ts),  yes.